crates/db_schema/src/source/captcha_answer.rs

```diff
@@ -0,0 +1,22 @@
+#[cfg(feature = "full")]
+use crate::schema::captcha_answer;
+use serde::{Deserialize, Serialize};
+use serde_with::skip_serializing_none;
+use uuid::Uuid;
+
+#[skip_serializing_none]
+#[derive(Clone, PartialEq, Eq, Debug, Serialize, Deserialize)]
+#[cfg_attr(feature = "full", derive(Queryable))]
+#[cfg_attr(feature = "full", diesel(table_name = captcha_answer))]
+pub struct CaptchaAnswer {
+  pub uuid: Uuid,
+  pub answer: String,
```

Comment:
Missing some fields. Check the other ones in the `source` folder for examples.

Reply:
pushed a commit which should resolve this and the above, it's ended up being a separate commit since there was a merge commit added and i didn't want to force push anything in